Python/Product/VSCode/AnalysisVsc/Program.cs

```diff
@@ -30,10 +34,15 @@ internal static class Program {
                 using (var cin = Console.OpenStandardInput())
                 using (var cout = Console.OpenStandardOutput())
                 using (var server = new LanguageServer())
-                using (var rpc = JsonRpc.Attach(cout, cin, server)) {
+                using (var rpc = new JsonRpc(cout, cin, server)) {
+                    var ui = new UIService(rpc);
```

Comment:
You have two `UIService` instances here.

Reply:
Good catch